corehq/ex-submodules/couchforms/management/commands/purge_forms_and_cases.py

```diff
@@ -0,0 +1,132 @@
+import csv
+
+from django.core.management.base import BaseCommand, CommandError
+from corehq.apps.app_manager.models import Application
+from casexml.apps.case.xform import get_case_ids_from_form
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors, CaseAccessors
+from dimagi.utils.django.management import are_you_sure
+
+
+X_FORM_HEADER = ['Version', 'Form Id', 'Domain']
+CASE_HEADER = ['CaseID', 'CaseType', 'CaseOpenedOn', 'CaseName', 'CaseUserId', 'CaseXFormIds']
+X_FORM_FILENAME = 'X-Form-Details.csv'
```

Comment:
super nit picky but "XForm" would be more consistent with naming everywhere else